### PR TITLE
Fix pagination-dropdown styles

### PR DESCRIPTION
**Pull request by riacataquian**
- [x] Add space between the letters
- [x] Align the `Sort By` dropdown to the right
- [x] Remove unused variables in `variables.scss`

In reference to [this](https://github.com/hexpm/hex_web/issues/313) issue.

Mobile view:
screenshot 2016-05-12 00 58 35

Web view:
screenshot 2016-05-12 01 23 27

### CODE REVIEW

File: web/static/css/base/_variables.scss [hunk 1]
```diff
@@ -3,15 +3,24 @@
 *********************************************************************/
 $color-white:         #ffffff;
 $color-black:         #000000;
-$color-red:           #a93e42;
+$color-red:           #aa2c3f;
```
Comment (pragmaticivan) [inline web/static/css/base/_variables.scss]: Why this color change?

**Comment by pragmaticivan** [comment]
Seems pretty good, what about this border in the left and right side for the desktop view? it's not in the current layout

File: web/static/css/base/_variables.scss (hunk 1 above)
Comment (riacataquian) [inline web/static/css/base/_variables.scss]: `$color-red` is an unused variable.

**Comment by riacataquian** [comment]
@pragmaticivan it was... an improvement suggestion. Meh? :D

**Comment by pragmaticivan** [comment]
That seems to be good for the mobile but not for the web, can you test this border just for mobile?

**Comment by pragmaticivan** [comment]
What do you think @ericmj ?

**Comment by riacataquian** [comment]
@pragmaticivan I'll revert the web view to the current layout. 👍

**Comment by ericmj** [comment]
Looks good, thank you!

**Pull request merged**